en/docs/learn/consume-api/manage-subscription/advanced-topics/adding-an-api-subscription-update-workflow.md

```diff
@@ -0,0 +1,54 @@
+# Adding an API Subscription Update Workflow
+
```

Comment:
Shall we  rename the title to Adding an API Subscription Tier Update workflow

Reply:
resolved